### PR TITLE
Add support for PHP 8.3

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,22 +10,22 @@
...
     "license": "BSD-3-Clause",
     "require": {
-        "php": "~8.1.0 || ~8.2.0",
-        "fig/http-message-util": "^1.1",
-        "mezzio/mezzio-router": "^3.1",
-        "psr/container": "^1.0 || ^2.0",
-        "psr/http-factory": "^1.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "fig/http-message-util": "^1.1.5",
+        "mezzio/mezzio-router": "^3.17",
```
Comment (gsteel) [inline composer.json]: @fezfez - Thank you, but can you please only bump dev deps unless there's a good reason to bump specific minimum deps in require. Bumping `require-dev` to latest with `composer bump -D` is absolutely fine.

**Review by gsteel — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (fezfez) [reply composer.json]: done

**Review by gsteel — APPROVED** [review]
Thanks @fezfez

**Comment by froschdesign** [comment]
@fezfez @gsteel 
A squashing of the commits should be done, otherwise meaningless commits (like "review") will land in the history.

**Pull request merged**